sphinx_scality/layout.html

```diff
@@ -111,15 +111,18 @@ <h2><a href="{{ pathto(pagename) }}">{{ title }}</a></h2>
       {%- endblock %} {# sidebarglobaltoc #}
 
       {%- block sidebarreleaseinfo %}
-      {%- if release | default('') != '' %}
       <div class="sidebar-relinfo">
-        {# TODO: add link to release notes and other hosted releases #}
+        {%- if release | default('') != '' %}
         <p>Release: <span class="value">{{ release }}</span></p>
+        {%- endif %}
+
+        {%- if theme_releasenoteslink | default('') != '' %}
+        <a href="{{ theme_releasenoteslink }}">Release Notes</a>
+        {%- endif %}
       </div>
-      {%- endif %}
       {%- endblock %} {# sidebarreleaseinfo #}
     </div>
-  {%- endblock %} {# sidebarleft #}
+      {%- endblock %} {# sidebarleft #}
```

Comment:
Why this indentation change?

Reply:
I thought I put it back like it was before ><

Reply:
And I actually lined it back with the opening one, should I revert? @gdemonet

Reply:
No, the opening one is L.89, and it was correctly indented before your change :wink: 
So yes, please revert, and then I'll approve :+1: